File: core/Core/Models/LoginLog.cs
using System;
using System.Collections.Generic;

namespace Core.Models;

public partial class LoginLog
{
    public int LogId { get; set; }

    public required int CustomerId { get; set; }

    public required DateTime LoginTime { get; set; }

    public required string IpAddress { get; set; }

    public required string DeviceInfo { get; set; }

    public required Customer Customer { get; set; }
}

File: analytic/domain/login_log/login_log.py
from datetime import datetime


class LoginLog:
    def __init__(self,
                 log_id: int,
                 customer_id: int,
                 login_time: datetime,
                 ip_address: str,
                 device_info: str):
        self.log_id = log_id
        self.customer_id = customer_id
        self.login_time = login_time
        self.ip_address = ip_address
        self.device_info = device_info